### ISSUE TITLE
Prepare view_run_batch page

### ISSUE THREAD

**Issue by jnak-cfe** [cfe-lab/Kive#568]
Summary: Post-completion of #536, #537, #538, we need a mutation of the view_runs page which displays all the runs in a specific batch.

My discussions with @rhliang have yielded a satisfactory data format for run_pipeline (see 108aadc5b58e97410e24a488b3e987eda396a309). Run_pipeline now needs to be able to process this data and redirect the user to a view_run_batch page.

This page would look similar to view_runs, only pre-filtered to only show the batch, and with extra form elements to edit the name and description of the batch as a whole. (Description is made same for every run in the batch, but name is independent. Initially, description is blank and name is set to the run name prefix field or a unique identifier if no prefix is set.)

Permissions are not editable on this page and are set on the previous page, choose_inputs.

**Comment by donkirkby** [comment]
I suggest we leave the run names blank by default, and change the run's display name to use the batch name plus first input if there is one, otherwise pipeline plus first input.

**Comment by rhliang** [comment]
As I was looking at how to do this, I noticed something that we should probably take care of.  Initially our thinking was to allow the user to start the batch and then edit name, description, and permissions.  Looking at the internals now, I'm reminded that the permissions defined on the Runs will affect the way they execute, because the permissions limit the number of ExecRecords that it can choose from on reuse.  This, I think, is why the View Results page for a single run doesn't allow you to change the metadata of Runs until they're finished.

We could talk about changing the way the execution occurs to suit this, but I don't see an obvious solution, and whatever we came up with would require a decent amount of internal changes and would certainly delay v0.8.

For now, anyway, I think it would be best if we put a permissions widget on the "start pipeline" page, for this reason.  After the runs are all finished, then we can allow them to change the permissions on the entire batch, but until then I think this should be disabled.
